src/Umbraco.Web.UI.Client/src/packages/search/examine-management-dashboard/views/section-view-examine-searchers.ts

```diff
@@ -34,6 +36,17 @@ export class UmbDashboardExamineSearcherElement extends UmbLitElement {
 	@state()
 	private _workspacePath = 'aa';
 
+	@state()
+	_totalPages = 1;
+
+	@state()
+	_currentPage = 1;
+
+	@state()
+	_total = 0;
```

Comment:
Maybe `_totalNumberOfResults` would be a bit clearer as to what this is the total of?